build/Dockerfile

```
@@ -307,8 +307,12 @@ apt-get install -y --no-install-recommends \
     libboost-filesystem1.81.0 \
     libboost-log1.81.0 \
     libcrypto++8 \
-    procps
+    procps \
+    nodejs \
+    npm
 rm -rf /var/lib/apt/lists/*
+npm install -g postgraphile
+npm install -g --save @graphile-contrib/pg-simplify-inflector
```

I suggest lock at least into a minor or major version.

```suggestion
npm install -g postgraphile@4.13
npm install -g --save @graphile-contrib/pg-simplify-inflector@6.1
```

If reproducibility of the container image is desired, put the full version.